glusterd2/volgen/store-utils.go

```diff
@@ -48,3 +48,14 @@ func GetVolfile(volfileID string) ([]byte, error) {
 	}
 	return resp.Kvs[0].Value, nil
 }
+
+// DeleteVolumeVolfiles deletes all the Volfiles belonging to a Volume
+func DeleteVolumeVolfiles(volname string) error {
+	_, err := store.Store.Txn(context.TODO()).If().
+		Then(
+			clientv3.OpDelete(volfilePrefix+volname, clientv3.WithPrefix()),
+			clientv3.OpDelete(volfilePrefix+"snaps/"+volname, clientv3.WithPrefix()),
```

Comment:
This code is okay but just a question: Volumes cannot be deleted without deleting snapshots first. If that's the case, snapshot volfiles should be deleted when snapshot itself is deleted, right ?

Reply:
yeah, you are right. I will remove `snaps/<volname>` from here.